src/btree/bt_slvg.c

```diff
@@ -676,7 +680,15 @@ __slvg_trk_leaf(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, uint8_t *ad
     case WT_PAGE_ROW_LEAF:
         WT_TIME_AGGREGATE_INIT_MERGE(&trk->trk_ta);
         WT_CELL_FOREACH_KV (session, dsk, unpack) {
-            WT_TIME_AGGREGATE_UPDATE(session, &trk->trk_ta, &unpack.tw);
+            switch (unpack.type) {
```

Comment:
Should we do anything here? I think we will automatically aggregate when we reconcile the page.

Reply:
I think this was already discussed offline that we need it.

Reply:
This is more complicated than I thought. By checking the code, we need to handle the size of overflow items and they may not be available in the corrupted data. I think we can ignore aggregating in salvage and let letter checkpoint to aggregate the count for us.